Source/Core/Core/HW/WiimoteReal/WiimoteReal.cpp

```diff
@@ -892,11 +892,13 @@ bool CheckForButtonPress(int wiimote_number)
   if (!g_wiimotes_mutex.try_lock())
```

Comment:
We should probably just use a scoped lock in the future, otherwise it's way too easy to forget a call to unlock().